code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -20,6 +20,7 @@ var/global/list/image/splatter_cache=list()
 	var/basecolor="#A10808" // Color when wet.
 	var/list/datum/disease2/disease/virus2 = list()
 	var/amount = 5
+	appearance_flags = NO_CLIENT_COLOR
```

Comment:
Would it be possible to do a check to see if the player is wearing the noir glasses and, if so, set appearance_flags |= ~NO_CLIENT_COLOUR-- otherwise, appearance_flags &= ~NO_CLIENT_COLOUR? Or does the system just not work that way? (Lol, I'm probably misunderstanding the way this bit here works. I haven't yet looked into it, and this is just a hunch.)